src/app/activity/bounty/Submit.ts

```diff
@@ -39,18 +41,26 @@ export const submitBounty = async (request: SubmitRequest): Promise<void> => {
 			messageId = getDbResult.dbBountyResult.discordMessageId;
 		}
 		const bountyChannel = await submittedByUser.client.channels.fetch(channelId) as TextChannel;
-		bountyEmbedMessage = await bountyChannel.messages.fetch(messageId).catch(e => {
+		submitterMessage = await bountyChannel.messages.fetch(messageId).catch(e => {
 			LogUtils.logError(`could not find bounty ${request.bountyId} in channel ${channelId} in guild ${request.guildId}`, e);
 			throw new RuntimeError(e);
 		});
     } else {
-        bountyEmbedMessage = request.message;
+        submitterMessage = request.message;
     }
 
+	if (getDbResult.dbBountyResult.creatorMessage !== undefined) {
+		const bountyChannel: TextChannel = await createdByUser.client.channels.fetch(getDbResult.dbBountyResult.creatorMessage.channelId) as TextChannel;
```

Comment:
You don't need to cast and declare a variable. The cast is sufficient if you need to coerce the type.